plugins/mcreator-core/features/feature_geode_simple.json

```diff
@@ -0,0 +1,97 @@
+{
+  "args0": [
+    {
+      "type": "field_number",
+      "name": "size",
+      "value": 1,
+      "min": 0.01,
+      "max": 1

```

Comment:
Does a scale larger than 1 not work? This way one could make large geodes.

Reply:
Bigger geodes are usually cut off when they generate, which is why I limited the size to 1

Reply:
I see